### PR TITLE
add pool caching

### PR DESCRIPTION
**Pull request by andreipope**
add pool caching

### CODE REVIEW

File: content/reference/CLI/service.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ description: How to use PX CLI service.
 weight: 16
 ---
 
+Portworx's CLI tool (`pxctl`) lets you run various service operations such as:
+
+- perform a node audit
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: Capitalize the start of bullet lists

```suggestion
- Perform a node audit
```

File: content/reference/CLI/service.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ description: How to use PX CLI service.
...
+- manage the call home feature
+- generate diagnostics package
+- get the version of the installed softwate
```
Comment (nathan-px) [inline content/reference/CLI/service.md]:
```suggestion
- Get the version of the installed software
```

File: content/reference/CLI/service.md [hunk 3]
```diff
@@ -6,6 +6,20 @@ description: How to use PX CLI service.
...
+- configure kvdb
+- place Portworx in maintenance mode
+- manage physical storage drives
+
+These commands are helpful when you want do debug issues related to your _Portworx_ cluster.
+
+You can get an overview of the available service operations by running:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: "get" is a pretty generic verb, I tend to favor something more descriptive to the action. In this case: "see"
```suggestion
You can see an overview of the available service operations by running:
```

File: content/reference/CLI/service.md [hunk 4]
```diff
@@ -34,46 +48,44 @@ OPTIONS:
    --help, -h  show help
 ```
 
-### pxctl service audit
+## How to Perform a Node Audit
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: These headlines are much improved. I think you can drop the "How to" portion of them
```suggestion
## Perform a Node Audit
```

File: content/reference/CLI/service.md [hunk 5]
```diff
@@ -34,46 +48,44 @@ OPTIONS:
...
 
-Audit the PX node
+You can audit the node with:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: A small detail, but is `pxctl` specific to Portworx nodes? 

```suggestion
You can audit the node with:
```

File: content/reference/CLI/service.md [hunk 6]
```diff
@@ -126,9 +138,10 @@ PX (OCI) Build Version:  1d83ac2baeb27451222edcd543249dd2c2f941e4
 PX Kernel Module Version:  72D3C244593F45167A6B49D
 ```
 
-### pxctl service kvdb
+## How to Configure kvdb
+
+You can configure kvdb with `pxctl service kvdb`. To get an overview of the available subcommands, run:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]:
```suggestion
You can configure the KVDB with `pxctl service kvdb`. To get an overview of the available subcommands, run:
```

File: content/reference/CLI/service.md [hunk 7]
```diff
@@ -147,9 +160,11 @@ OPTIONS:
    restore      Restore keys and values into kvdb from a kvdb.dump file
 ```
 
-### pxctl service maintenance
+## How to Place Portworx in Maintenance Mode
+
+Running `pxctl service maintenance ` will tell the cluster that it is going down for maintenance. Once the server is offline, you can add/remove drives, add memory, etc…
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: It's not clear to me what "to tell the cluster that it is going down for maintenance" is referring to. Does this instruct Portworx to place the cluster in a maintenance state? 

```suggestion
Run the `pxctl service maintenance` command to tell the cluster that it is going down for maintenance. Once the server is offline, you can add/remove drives, add memory, etc…
```

File: content/reference/CLI/service.md [hunk 8]
```diff
@@ -228,9 +243,9 @@ Drive add  successful. Requires restart (Exit maintenance mode).
 
 To rebalance the storage across the drives, use pxctl service drive rebalance. This is useful after prolonged operation of a node.
 
-### pxctl service drive show
+## How to Display Drive Information
 
-You can use pxctl service drive show to display drive information on the server
+You can use `pxctl service drive show` to display drive information on the server:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]:
```suggestion
You can use the `pxctl service drive show` command to display drive information on the server:
```

File: content/reference/CLI/service.md [hunk 9]
```diff
@@ -239,17 +254,17 @@ pxctl service drive show
...
+    Has meta data: Yes
+    Drives:
+    1: /dev/mapper/volume-e85a42ca, 1.0 GiB allocated of 100 GiB, Online
 ```
 
-### pxctl service email
+## How to Configure Email Settings for Alerts
 
-Email setting commands
+Here's how you can get the list of available subcommands:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]: The language here isn't consistent with the language used in the other steps:

```suggestion
You can use the `pxctl service email` command to list the available subcommands:
```

File: content/reference/CLI/service.md [hunk 10]
```diff
@@ -355,9 +370,9 @@ COMMANDS:
    update    Update pool properties
 ```
 
-### pxctl service pool update
+## How to Update Pool Properties
 
-Updates the pool properties
+`pxctl service pool update` lets you perform several operations such as resizing a pool, setting the IO priority or adding labels:
```
Comment (nathan-px) [inline content/reference/CLI/service.md]:
```suggestion
the `pxctl service pool update` command lets you perform several operations including the following: 
* Resize a pool
* Set the IO priority
* Add labels
```

File: content/reference/CLI/service.md [hunk 11]
```diff
@@ -6,6 +6,20 @@ description: How to use PX CLI service.
 weight: 16
 ---
 
+Portworx's CLI tool (`pxctl`) lets you run various service operations such as:
```
[the remark this answers is not in the document]
Comment (andreipope) [reply content/reference/CLI/service.md]: @nathan-px solved

File: content/reference/CLI/service.md (hunk 6 above)
Comment (andreipope) [reply content/reference/CLI/service.md]: @nathan-px Probably better to create a new JIRA.

File: content/reference/CLI/service.md (hunk 7 above)
Comment (andreipope) [reply content/reference/CLI/service.md]: @nathan-px Probably better to create a new JIRA.

File: content/reference/CLI/service.md (hunk 5 above)
Comment (andreipope) [reply content/reference/CLI/service.md]: @nathan-px Yes, it's the CLI tool

File: content/reference/CLI/service.md (hunk 7 above)
Comment (andreipope) [reply content/reference/CLI/service.md]: @nathan-px Yes, I think so.

**Pull request closed without merging**